src/web/components/ClientSideTokenGeneration/CstgDomain.tsx

```diff
@@ -3,30 +3,28 @@ import { useState } from 'react';
 
 import { Dialog } from '../Core/Dialog';
 import { TriStateCheckbox } from '../Core/TriStateCheckbox';
+import CstgEditDomainDialog from './CstgEditDomainDialog';
 
-type DeleteConfirmationDialogProps = {
+type DeleteConfirmationDialogProps = Readonly<{
   domain: string;
   onRemoveDomain: () => void;
-};
+  onOpenChange: () => void;
+}>;
 
-function DeleteConfirmationDialog({ domain, onRemoveDomain }: DeleteConfirmationDialogProps) {
-  const [openConfirmation, setOpenConfirmation] = useState(false);
-
-  const handleRemove = async () => {
-    setOpenConfirmation(false);
-    await onRemoveDomain();
+function DeleteConfirmationDialog({
```

Comment:
There's now separate files for adding and editing domains. Perhaps now is a good time to also extract the delete dialog component to a separate file, for consistency?

Reply:
Then this file could be renamed `CstgDomainItem.tsx`

Reply:
Done